package.json

```diff
@@ -9,18 +9,22 @@
     "lint": "next lint"
   },
   "dependencies": {
+    "@hookform/resolvers": "^3.6.0",
+    "next": "^14.2.4",
+    "random-words": "^2.0.1",
```

Comment:
i would like to not have to use this library, but for now this is a quick way to get an expansive word list.